docs/source/run/plugin.rst

```diff
@@ -0,0 +1,137 @@
+==============
+
+Install single file plugins
+---------------------------
+
+To add a single file plugin to your Sopel instance, copy the file into the
+bot's ``plugins`` directory under the
+:attr:`~sopel.config.core_section.CoreSection.homedir` directory. By default,
+the plugin file should be copied into ``~/.sopel/plugins``.
+
+.. note::
+
+    Following the :doc:`systemd service example <service>`, if your homedir is
+    ``/var/lib/sopel`` then your plugin directory is
+    ``/var/lib/sopel/plugins``.
+
+Start (or restart) your bot and Sopel will load the plugin automatically.
+
+Install packaged plugins
+------------------------
+
+Plugin authors might also publish their works as packages; you can find them by
+searching PyPI, or by using your favorite search engine to search for
+e.g. "sopel reminder plugin".
+
+You can install these packages with ``pip``, the same way it is recommended to
+install Sopel. All you need is to install the plugin for the same Python
+interpreter.
+
+For example, you can install the "remind" plugin from the ``sopel-remind``
+package::
+
+    python3 -m pip install sopel-remind
+
+This will install the plugin from PyPI where you can see `all the releases`__
+of this package.
+
+.. note::
+
+    Each plugin can have specific system dependencies or other requirements
+    that you need to take care of when installing it. Refer to the plugin's
+    documentation for more details.
+
+.. __: https://pypi.org/project/sopel-remind/
+
+
+Enabling or disabling plugins
+=============================
+
+By default, any plugins that Sopel can load will be automatically enabled: it
+is true for all types of plugins (built-in, single file, or packaged plugins).
+You can control this behavior by disabling some plugins, or enabling only some
+plugins.
+
+You can disable a plugin by using the ``sopel-plugins disable`` command line,
+e.g. to disable the remind plugin mentioned before::
+
+    sopel-plugins disable remind
+
+This will add the plugin to the
+:attr:`~sopel.config.core_section.CoreSection.exclude` list. The next time the
+bot starts (or restarts), the plugin won't be loaded.
+
+You can re-enable a plugin by using the ``sopel-plugins enable`` command line,
+e.g. to re-enable the remind plugin::
+
+    sopel-plugins enable remind
+
+Which will remove the plugin from the ``exclude`` list. The plugin will be
+loaded as before the next time the bot starts (or restarts).
+
+Enable-only plugins
+-------------------
+
+By default, all plugins are enabled, and you disable plugins one by one.
+However, you may want to enable only a set of plugins, and disable every other
+one by default.
+
+To do so, the enable command has a ``--allow-only`` option: this will add the
+plugin to the :attr:`~sopel.config.core_section.CoreSection.enable` list::
+
+    sopel-plugins enable --allow-only remind
+
+When you start the bot (or restart), the bot will load the plugins from the
+``enable`` list, and **only these plugins**.
+
```

Comment:
Disclaimer up front: This might be too much detail. An alternative would be mentioning the `-a` switch as a shortcut to typing out the whole `--enable-only` name all the time.

```suggestion

.. note::

    You only need to use the ``--enable-only`` option once. If the ``enable``
    list is not empty, ``sopel-plugins enable`` and ``sopel-plugins disable``
    will add and remove plugins in that list.

```